hs/src/Reach/AST/DLBase.hs

```diff
@@ -398,7 +404,10 @@ instance Pretty DLExpr where
       DLE_MapSet _ mv kv nv ->
         pretty mv <> "[" <> pretty kv <> "]" <+> "=" <+> pretty nv
       DLE_MapDel _ mv i -> "delete" <+> pretty mv <> brackets (pretty i)
-      DLE_Remote _ _ av m amta as -> "remote(" <> pretty av <> ")." <> viaShow m <> ".pay" <> parens (pretty amta) <> parens (render_das as)
+      DLE_Remote _ _ av m amta as bill (DLWithBill _ nonNetTokRecv _) ->
```

Comment:
I'm still reading, but I don't understand the `bill` field

Reply:
`bill` is the non-network amounts/tokens provided when the user calls `remote.bill( AMTS )`. Since the network amount is already handled, I just passed the non-network stuff along to solidity instead of passing a `DLPayAmt` object. 

`DLWithBill ` is used to represent the tokens we should/should not receive via `withBill`.